WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingViewModel.swift

```diff
@@ -172,6 +172,18 @@ private extension StoreOnboardingViewModel {
                         } else {
                             return true
                         }
+                    }).map({ [weak self] task in
+                        // If store is already live and launchStore task is incomplete
+                        // mark the task as complete
```

Comment:
nit: might be worth mentioning that this is a workaround for a backend issue, in case any developer wonders why the launch store task needs special handling

Reply:
Added more details and link to GH issue in 496a7ebef0c13ca5db65933d1998d61a298d9ba6